src/components/ConnectionStatus.svelte

```diff
@@ -10,9 +11,11 @@
   <!-- TODO: ADD REAL DATA WHEN STORE IS CONNECTED -->
   <div class="flex items-center gap-2">
     <div class="w-1 h-1 rounded-full bg-green-success p-1" />
-    <p class="text-md uppercase">name_OF_NETWORK</p>
+    <p class="text-md uppercase">{$selectedNetwork?.name}</p>
   </div>
-  <p class="text-md">NETWORK_ENDPOINT_URL</p>
+  <p class="text-md">
+    {#if $selectedNetwork}{$selectedNetwork.endpoint?.toString().replace(/\/$/, '')}{/if}
+  </p>
```

Comment:
As this is defined in a different issue (#106 ), please either split it up into a PR for the defined issue or make clear that #106 is being closed here.

Reply:
It still needs some work.  I'm going to keep it in this PR and finish/close it in another PR.